.github/workflows/pages.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - name: Generate Index
         shell: bash
         run: |
-          find . -name javadoc -type d -print > index.txt
+          find . -name javadoc -type d -print | sort --version-sort --reverse > index.txt
```

Comment:
I don't think this workflow is used any longer... You will probably want to change the `find` line in publish-javadoc's last step instead.

And yeah this file should be removed. This one was to make it publish after the other actions made a commit. Turns out github doesn't run workflows by a commit pushed from github actions (except deploy to github one), so eventually I ditched the action based github pages publish route and went back to the old style.